### ISSUE TITLE
MobileNetV3 backbone support

### ISSUE THREAD

**Issue by moonsh** [tobibaum/metrabs_trt#5]
Thank you for sharing this work. It helps me a lot.

I am trying to use mobilenetv3 backbone but I got some weight size mismatching problems.
I used the below additional script for mobilenetv3. Is there something I didn't include ?

elif model_name.startswith('mobilenet'):
    from options import FLAGS
    FLAGS.centered_stride = True
    bn = keras.layers.BatchNormalization

    import backbones.mobilenet_v3
    class MyLayers(backbones.mobilenet_v3.VersionAwareLayers):
        def __getattr__(self, name):
            if name == 'BatchNormalization':
                return bn
            return super().__getattr__(name)

    arch = model_name
    arch = arch[:-4] if arch.endswith('mini') else arch
    classname = f'MobileNet{arch[len("mobilenet"):]}
    mod = getattr(backbones.mobilenet_v3, classname)(
        include_top=False, weights=None, minimalistic=False,
        input_shape=(None, None, 3), layers=MyLayers(),
        centered_stride=FLAGS.centered_stride, pooling=None)

**Comment by Tikitaka02** [comment]
I am also trying to do it with mobilenet. Any luck?
